src/utils/endpoints.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+function fetchAllArticles(){
+
+    return axios.get('https://backend-project-2yjd.onrender.com/api/articles').then((response) => {
+        return(response.data.articles);
+    })
+    .catch((err) => {
```

Comment:
Instead of logging here, please render a message conditionally to the user in the component.